crates/conjure_core/src/solver/adaptors/minion.rs

```diff
@@ -248,7 +248,40 @@ fn parse_exprs(
     minion_model: &mut MinionModel,
 ) -> Result<(), SolverError> {
     for expr in conjure_model.get_constraints_vec().iter() {
-        parse_expr(expr.to_owned(), minion_model)?;
+        // TODO: top level false / trues should not go to the solver to begin with
+        // ... but changing this at this stage would require rewriting the tester
+        use crate::metadata::Metadata;
+        use conjure_ast::Constant;
+        use conjure_ast::Expression as Expr;
+
+        match expr {
+            // top level false
+            Expr::Constant(_, Constant::Bool(false)) => {
+                parse_expr(
+                    Expr::Eq(
+                        Metadata::new(),
+                        Box::new(Expr::Constant(Metadata::new(), Constant::Int(1))),
```

Comment:
is this really the best way of doing top level true/false for minion @ChrisJefferson ?

Reply:
As per https://github.com/conjure-cp/conjure-oxide/issues/366#issuecomment-2412680690, 

Minion has true() and false()